daemon/info.go

```diff
@@ -171,6 +172,32 @@ func (daemon *Daemon) fillSecurityOptions(v *types.Info, sysInfo *sysinfo.SysInf
 	v.SecurityOptions = securityOptions
 }
 
+func (daemon *Daemon) fillAPIInfo(v *types.Info) {
+	const warn string = `
+         Access to the remote API is equivalent to root access on the host. Refer
+         to the 'Docker daemon attack surface' section in the documentation for
+         more information: https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface`
+
+	cfg := daemon.configStore
+	for _, host := range cfg.Hosts {
+		// cnf.Hosts is normalized during startup, so should always have a scheme/proto
+		h := strings.SplitN(host, "://", 2)
```

Comment:
Probably fine for this PR, but it would be nice to eliminate this by being able to do something like `host.Protocol()`

Reply:
Yes; tried to keep this PR minimal, but I'm thinking of exposing some of this information (e.g., the configured hosts) in `docker info` as well. I can do some refactoring as part of that 👍